frontend/src/pages/Login.tsx

```diff
@@ -1,7 +1,280 @@
-type LoginProps = {
-  title?: string;
+import { useState } from "react";
+import type { ChangeEvent, FormEvent } from "react";
+import '../css/LoginPage.css';
+import opentechlogo from "../img/opentechlogo.png";
+import craftmergelogo from "../img/craftmergelogo.png";
+import craftmergelogoblack from "../img/craftmergelogoblack.png";
```

Comment:
Images are imported and used in the UI, but there is no fallback or error handling if an image fails to load, risking broken visuals.

Add fallback logic or alt text for images; consider error boundaries or placeholders.